docker/gaffer-jhub-options-server/conf/graphs.yaml

```diff
@@ -0,0 +1,19 @@
+- id: simple-graph
+  name: Simple Graph
+  graphConfig:
+    description: "A very simple graph"
+  restApi: http://gaffer-api.gaffer.svc:80/rest/latest
+- id: road-traffic
+  name: Road Traffic Example Graph
+  graphConfig:
+    description: "A graph containing sample GB road traffic data from the Department of Transport"
+  restApi: http://road-traffic-gaffer-api.gaffer.svc:80/rest/latest
```

Comment:
I tried running this locally and couldn't find the road traffic example at this url, am I missing something?

Reply:
I got this working now. In Python the url can be found using `os.getenv("GAFFER_REST_API_URL")`. Perhaps this should be documented?